pr_review_list.rb

```diff
@@ -113,3 +115,4 @@ def does_array_have_pr(array, pr_number)
 File.open('report.json', 'wb') do |f|
   JSON.dump(open_prs, f)
 end
+end
```

Comment:
See my comment above about the indentation

Reply:
Please do not mark resolved until it is @Disha-maker

Reply:
Ok, I will fix the indentation :)